Source/Core/InputCommon/GCAdapter.cpp

```diff
@@ -87,10 +87,14 @@ static std::array<u8, SerialInterface::MAX_SI_CHANNELS> s_controller_type = {
     ControllerTypes::CONTROLLER_NONE, ControllerTypes::CONTROLLER_NONE};
 static std::array<u8, SerialInterface::MAX_SI_CHANNELS> s_controller_rumble{};
 
+constexpr size_t CONTROLER_INPUT_PAYLOAD_EXPECTED_SIZE = 37;
```

Comment:
Does it make sense to move these above their respective usages (at least until there are extra scenarios where they are used) so as to improve readability?

Reply:
They're already used in several places (both for declaring the variables and when reading into them), and IMO it's better to have them all together to highlight that the sizes are different (and that different sizes are used for rumble and init).